frontend/src/theme/components/button.ts

```diff
@@ -121,6 +121,13 @@ const Button = {
       fontSize: "xs",
       height: "32px",
     },
+    transparent: {
+      bg: "transparent",
+      padding: "0px",
+      margin: "0px",
+      border: "0px",
+      fontWeight: "normal",
+    },
```

Comment:
Basically used this as a way to avoid it going to default styling. Not sure if there's something similar that already exists?